crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -33,6 +33,9 @@ class ApplicationController < ActionController::Base
     Crowbar::Installer.successful? || \
     Rails.env.test?
   }
+  before_filter :upgrade, if: proc {
+    File.exist?("/var/lib/crowbar/upgrade/progress.yml")
```

Comment:
I'd prefer a solution which will work cleanly if we use a similar mechanism for 7->8 upgrades.  The current file gives no indication of which product version it refers to.  So it will lie around for a long time until 7->8 upgrade, and then what happens?  OTOH, we could have it called

    /var/lib/crowbar/upgrade/6-to-7-progress.yml

and then on completion it could be renamed to

    /var/lib/crowbar/upgrade/6-to-7-completed.yml

This would allow the same quick file existence tests but also make it future-proof.